Simulator/Core/Driver.cpp

```diff
@@ -207,7 +212,10 @@ bool parseCommandLine(int argc, char *argv[]) {
        || (cl.addParam("deserializefile", 'r', ParamContainer::filename,
                        "simulation deserialization filepath (enables deserialization)") != ParamContainer::errOk)
        || (cl.addParam("serializefile", 'w', ParamContainer::filename,
-                       "simulation serialization filepath (enables serialization)") != ParamContainer::errOk)) {
+                       "simulation serialization filepath (enables serialization)") != ParamContainer::errOk)
+       || (cl.addParam("version", 'v', ParamContainer::novalue,
+                       "current git commit ID") != ParamContainer::errOk)) {
```

Comment:
I hate to do this, but the docstring should be "output current git commit ID and exit".